simpleflow/canvas.py

```diff
@@ -3,6 +3,7 @@
 from .base import Submittable
 from .signal import WaitForSignal
 from .task import ActivityTask, SignalTask
+from .step import Step
 
 
 def propagate_attribute(obj, attr, val):
```

Comment:
Good idea, I tried to make make it inheriting from `Group` but the activities were directly transformed as ActivityTask and that not that's we wanted.

Can steps raises_on_failure? Yes for the activites inside, not inside ?